django_project/analysis/analysis.py

```diff
@@ -678,6 +678,7 @@ def run_analysis(locations: list, analysis_dict: dict, *args, **kwargs):
         return analysis_cache.create_analysis_cache(reduced.getInfo())
 
     if analysis_dict['analysisType'] == "Baseline":
+        print("Running baseline analysis")
```

Comment:
@osundwajeff please remove all print calls, if you need them, please use logger instead